packages/manager/modules/pci/src/projects/project/workflow/add/resources/resources.controller.js

```diff
@@ -49,4 +50,8 @@ export default class {
   selectResource(instance) {
     this.selectedResource = instance;
   }
+
+  toFlavors() {
+    return new Flavors(this.selectedResource.flavor);
```

Comment:
As `selectedResource` is an instance, why implement this method in controller instead of instance class? (to have something  reusable, eg `instance.isFlavorType(IOPS)` which behind check the type of the instance flavor)

Reply:
I made the changes to allow reuse of type tests